### PR TITLE
Bump SIP to 4.19.17 & PyQt to 5.12.2

### PR DESCRIPTION
**Pull request by Lctrs**

### CODE REVIEW

**Comment by Lctrs** [comment]
@bochecha Build is failing because file `Python.h` can't be found anymore. Could be a bug in `freedesktop-sdk` ?

**Comment by bochecha** [comment]
```
g++ -c -pipe -O2 -fno-exceptions -Wall -W -D_REENTRANT -fPIC -DSIP_PROTECTED_IS_PUBLIC -Dprotected=public -DPy_LIMITED_API=0x03040000 -DQT_NO_EXCEPTIONS -DQT_NO_DEBUG -DQT_PLUGIN -DQT_CORE_LIB -I. -I. -isystem /app/include/python3.7m -I../qpy/QtCore -isystem /usr/include/QtCore -I. -I/usr/lib/mkspecs/linux-g++ -o qpycore_types.o ../qpy/QtCore/qpycore_types.cpp
[… snip …]
../qpy/QtCore/qpycore_types.cpp:21:10: fatal error: Python.h: No such file or directory
 #include <Python.h>
          ^~~~~~~~~~
compilation terminated.
```

The command line has `isystem /app/include/python3.7m`, it should be `-isystem /usr/include/python3.7m` instead, as Python isin the SDK not the app.

That's either a bug in their buildsystem, or there's an option you need to add with the new version?

File: pyqt-configure [hunk 1]
```diff
@@ -1,9 +1,12 @@
 #!/bin/sh
 
+PYTHON_VERSION=$(${PYTHON} -c 'import sys; print(sys.version[:3])')
+
 ${PYTHON} configure.py \
     --assume-shared \
     --concatenate \
     --confirm-license \
+    --destdir=/app/lib/python${PYTHON_VERSION}/site-packages \
```
Comment (bochecha) [inline pyqt-configure]: That's a pretty terrible hardcoding here. :confused: 

You're assuming that it's a path added to Python's sys.path in the runtime.

I wish you hadn't merged that.

Comment (Lctrs) [reply pyqt-configure]: We're already doing the same hardcoding thing for sip anyway... I couldn't find another way of doing it.

**Pull request merged**